python/requirements.txt

```diff
@@ -5,7 +5,7 @@ blessings == 1.7
 distro == 1.4
 mozinfo == 1.2.1
 mozlog == 7.1.0
-setuptools == 65.5.1
```

Comment:
Great find!

I knew I need to update setuptools but didn't know that my every try to do that was overridden here.